src/instruments/template/template.js

```diff
@@ -1,53 +0,0 @@
-'use strict';
-
-/* global BaseInstrument */
-/* global registerInstrument */
-
-// eslint-disable-next-line camelcase
-class A32NX_INSTRUMENT_NAME_Logic extends BaseInstrument {
-    get templateID() {
-        return 'A32NX_INSTRUMENT_NAME_TEMPLATE';
-    }
-
-    get isInteractive() {
-        // eslint-disable-next-line
-        return INSTRUMENT_IS_INTERACTIVE;
-    }
-
-    get IsGlassCockpit() {
-        return true;
-    }
-
-    connectedCallback() {
-        super.connectedCallback();
-
-        // This is big hack, see `template.html`.
-        {
-            const code = document.getElementById('A32NX_BUNDLED_STYLE').innerHTML;
-            const style = document.createElement('style');
-            style.innerHTML = code;
-            document.head.appendChild(style);
-        }
-        {
-            const code = document.getElementById('A32NX_BUNDLED_LOGIC').innerHTML;
-            const script = document.createElement('script');
-            script.innerHTML = code;
-            document.body.appendChild(script);
-        }
-    }
-
-    Update() {
-        super.Update();
-        if (this.CanUpdate()) {
-            this.dispatchEvent(new CustomEvent('update', { detail: this.deltaTime }));
```

Comment:
This line is no longer current. We discovered in #3270 that BaseInstrument.deltaTime did not correctly take the glass cockpit refresh rate into account, thereby causing timing issues when the SimVar framework relied on this.

This has since been fixed in master here:

https://github.com/flybywiresim/a32nx/blob/7da6cf1c4290d17f737df6807754bb270ee01309/src/instruments/template/template.js#L11-L18

https://github.com/flybywiresim/a32nx/blob/7da6cf1c4290d17f737df6807754bb270ee01309/src/instruments/template/template.js#L55

...however, the rollup-plugin-msfs does not have that fix yet. So by merging this you will be effectively reverting that change and breaking the deltaTime logic. Could you push another release to rollup-plugin-msfs with the updated deltaTime logic?

Reply:
Fixed.